component/pom.xml

```diff
@@ -57,38 +58,38 @@
             <artifactId>org.wso2.carbon.datasource.core</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
+            <groupId>org.awaitility</groupId>
```

Comment:
Are we removing testng dependency? Also these needs to be propagated to parent pom

Reply:
No, these are just reordered to see clearly which has test scope